engine/rpcserver.go

```diff
@@ -1905,7 +1905,7 @@ func (s *RPCServer) GetAuditEvent(_ context.Context, r *gctrpc.GetAuditEventRequ
 
 // GetHistoricCandles returns historical candles for a given exchange
 func (s *RPCServer) GetHistoricCandles(_ context.Context, r *gctrpc.GetHistoricCandlesRequest) (*gctrpc.GetHistoricCandlesResponse, error) {
-	if r.Start >= r.End {
+	if r.Start == r.End {
```

Comment:
You are still comparing strings here and its not a good way of checking if the time is invalid. Please use the  `UTCStartTime` and `UTCEndTime` to validate the times with .Equal and .After